src/include/misc.h

```diff
@@ -48,6 +48,9 @@
 #define	WT_ALIGN(n, v)							\
 	((((uintmax_t)(n)) + ((v) - 1)) & ~(((uintmax_t)(v)) - 1))
 
+#define	WT_ALIGN_NEAREST(n, v)						\
+	((((uintmax_t)(n)) + ((v) / 2 + 1)) & ~(((uintmax_t)(v)) - 1))
+
```

Comment:
@agorrod, I can't imagine it has any practical significance, but shouldn't this be:
```
((((uintmax_t)(n)) + ((v) / 2)) & ~(((uintmax_t)(v)) - 1))
```

Hurts my head to think about it, so here's the test program:
```
% cat t.c
#include <inttypes.h>
#include <stdio.h>
#include <stdlib.h>

#define	WT_ALIGN_NEAREST_ORIG(n, v)					\
    ((((uintmax_t)(n)) + ((v) / 2 + 1)) & ~(((uintmax_t)(v)) - 1))
#define	WT_ALIGN_NEAREST_FIX(n, v)					\
    ((((uintmax_t)(n)) + ((v) / 2)) & ~(((uintmax_t)(v)) - 1))

int main(int argc, char **argv)
{
	int a;

	a = atoi(argv[1]);
	printf("%d -> orig %d, fix %d\n", a,
	    (int)WT_ALIGN_NEAREST_ORIG(a, 1024),
	    (int)WT_ALIGN_NEAREST_FIX(a, 1024));
	return 0;
}
% cc t.c
% ./a.out 510
510 -> orig 0, fix 0
% ./a.out 511
511 -> orig 1024, fix 0
% ./a.out 512
512 -> orig 1024, fix 1024
% ./a.out 513
513 -> orig 1024, fix 1024
```